user-org-es-util/src/main/java/org/sunbird/dto/SearchDtoMapper.java

```diff
@@ -0,0 +1,166 @@
+package org.sunbird.dto;
+
+import org.sunbird.dto.SearchDTO;
+
+import java.math.BigInteger;
+import java.util.*;
+
+/**
+ * This class will be used to map the SearchDto
+ *
+ * @author anmolgupta
+ */
+
+
+public class SearchDtoMapper {
+
+
+    /**
+     * this method will create SearchDto pojo object.
+     * @param searchQueryMap
+     * @return
+     */
+    public static SearchDTO createSearchDto(Map<String, Object> searchQueryMap) {
```

Comment:
Dislike static methods...Create a factory/mapper class that takes in Map and return a SearchDTO.